backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -824,6 +825,13 @@ export class UserResolver {
 
     return true
   }
+
+  @Authorized([RIGHTS.USER])
+  @Query(() => User, { nullable: true })
+  async user(@Arg('identifier') identifier: string): Promise<User | null> {
+    const user = await findUserByIdentifier(identifier)
+    return user ? new User(user) : null
+  }
```

Comment:
I suggest to always return a user or throw.
```suggestion
  @Query(() => User)
  async user(@Arg('identifier') identifier: string): Promise<User> {
    return new User(await findUserByIdentifier(identifier))
  }
```